script.js

```diff
@@ -1,8 +1,80 @@
-function myAdd(a, b) {
-  let c = 0;
-  c = a + b;
+const hamBurger = document.querySelector('.hamburger');
+const mobileMenu = document.querySelector('.hamburger-menu');
 
-  return c;
+hamBurger.addEventListener('click', () => {
+  hamBurger.classList.toggle('is-active');
+  mobileMenu.classList.toggle('is-active');
+});
+
+// creating first card item i.e grid project
+
+const projects = [{
+  chess: './images/chess-bg.png',
+  image: './images/chinedu-ezea-picture.png',
+  name: 'Chinedu Ezea',
+  dash: '',
+  qualification: 'Chinedu is a doctor of philosophy at Nnamdi Azikiwe University',
+  description: 'He is a mathematician and a professional editors with several publications in high impact factor journals',
+},
+{
+  chess: './images/chess-bg.png',
+  image: './images/chinedu-ezea-picture.png',
+  name: 'Chinedu Ezea',
+  dash: '',
+  qualification: 'Chinedu is a doctor of philosophy at Nnamdi Azikiwe University',
+  description: 'He is a mathematician and a professional editors with several publications in high impact factor journals',
+},
+{
+  chess: './images/chess-bg.png',
+  image: './images/chinedu-ezea-picture.png',
+  name: 'Chinedu Ezea',
+  dash: '',
+  qualification: 'Chinedu is a doctor of philosophy at Nnamdi Azikiwe University',
+  description: 'He is a mathematician and a professional editors with several publications in high impact factor journals',
+},
+{
+  chess: './images/chess-bg.png',
+  image: './images/chinedu-ezea-picture.png',
+  name: 'Chinedu Ezea',
+  dash: '',
+  qualification: 'Chinedu is a doctor of philosophy at Nnamdi Azikiwe University',
+  description: 'He is a mathematician and a professional editors with several publications in high impact factor journals',
+},
+{
+  chess: './images/chess-bg.png',
+  image: './images/chinedu-ezea-picture.png',
+  name: 'Chinedu Ezea',
+  dash: '',
+  qualification: 'Chinedu is a doctor of philosophy at Nnamdi Azikiwe University',
+  description: 'He is a mathematician and a professional editors with several publications in high impact factor journals',
+},
+{
+  chess: './images/chess-bg.png',
+  image: './images/chinedu-ezea-picture.png',
+  name: 'Chinedu Ezea',
+  dash: '',
+  qualification: 'Chinedu is a doctor of philosophy at Nnamdi Azikiwe University',
+  description: 'He is a mathematician and a professional editors with several publications in high impact factor journals',
+},
+];
+
+const sectionFeaturedElement = document.getElementById('featured-speakers-id');
+sectionFeaturedElement.className = 'featured-speakers';
+
+for (let i = 0; i < projects.length; i += 1) {
```

Comment:
- [ ] [Optional] You can add your code inside a function and use it, that way you will not repeat your code when you need to use it again in another place

Reply:
Hi Ismailco,

These changes you requested are not necessary. The reason is that these lines of code appear only once in my script.js file. A function is only required if the code appears more than once.